.github/workflows/ci.yml

```
@@ -0,0 +1,80 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
```

I think this could be more like
```suggestion
  pull_request:
    branches:
      - master
```

so that every PR on master runs lint and test rather than on every push to master.

---

Yeah, we can try your suggestion and see what it does. 👍

I think `pull_request:` (empty right-hand side) means every PR, which should include every PR on `master` branch? The current `push` configuration means the workflow runs when we merge PR to `master`.